src/Functions.UnitTests/Handlers/WhenApprenticeshipCreated.cs

```diff
@@ -51,5 +51,36 @@ public async Task ThenApprovalIsCreated()
                 @event.AccountLegalEntityId, 
                 @event.TrainingCourseVersion));
         }
+
+        [TestCase(null)]
+        [TestCase(false)]
+        public async Task ForSLDFundedApprenticeshipThenApprovalNotCreated(bool? isOnFlexiPaymentPilot)
+        {
+            var @event = _fixture.Build<ApprenticeshipCreatedEvent>().With(x => x.IsOnFlexiPaymentPilot, isOnFlexiPaymentPilot)
+                .Create();
+
+            await _handler.Handle(@event, new Mock<ILogger>().Object);
+
+            _apprenticeshipService.Verify(x => x.CreateApproval(
+                @event.Uln,
+                @event.FirstName,
+                @event.LastName,
+                @event.ApprenticeshipId,
+                @event.ProviderId,
+                @event.AccountId,
+                @event.LegalEntityName,
+                @event.EndDate,
+                @event.TransferSenderId,
+                @event.ApprenticeshipEmployerTypeOnApproval,
+                @event.PriceEpisodes,
+                @event.TrainingCode,
+                @event.DateOfBirth,
+                @event.StartDate,
+                @event.ActualStartDate,
+                @event.IsOnFlexiPaymentPilot,
+                @event.ApprenticeshipHashedId,
+                @event.AccountLegalEntityId,
+                @event.TrainingCourseVersion), Times.Never);
```

Comment:
Approved because this comment is so minor but could use It.IsAny<EventType> here since we want to verify CreateApproval is never called at all.